config/locales/en.yml

```diff
@@ -1179,6 +1191,13 @@ en:
         date_example: For example, 30 1 2017
       form:
         uploaded_request_files: Upload relevant files (optional)
+    ico_foi: 
+      recored_further_action:
+        heading: Further ICO request
```

Comment:
Not sure about what this is, was thinking maybe "Secondary ICO request"? Or "Related ICO request"?

Reply:
it not secondary , it is just more requests from ICO after ICO reviewed the case.   I just followed the design, maybe discuss with Ryan or Nicols ?